scribejava-apis/src/test/java/com/github/scribejava/apis/examples/OdnoklassnikiExample.java

```diff
@@ -18,14 +18,13 @@
 
     public static void main(String... args) {
         // Replace these with your client id and secret
-        final String clientId = "your client id";
-        final String publicKey = "your api secret";
-        final String clientSecret = "your client secret";
+        final String clientId  = "your api client id";
+        final String publicKey = "your api public key";
```

Comment:
BTW, Seems like it's OK-specific stuff. And it's correct name  is 'applicationKey', right?